eureka2-core/src/main/java/com/netflix/eureka2/interests/ChangeNotifications.java

```diff
@@ -37,4 +43,13 @@
         }
         return Observable.from(notifications);
     }
+
+    public static Func1<ChangeNotification<?>, Boolean> batchMarkerFilter() {
```

Comment:
This filter returns true for data notification types, so it would make more sense for the name to be dataNotificationFilter (or related to the data types instead of batch markers).